ui/src/views/Stacktrader/Stacktrader.js

```diff
@@ -29,12 +29,12 @@ class Stacktrader extends Component {
        */
       entity_id: "",
       shard: "",
-      position: { x: 0.0, y: 0.0, z: 0.0 },
+      position: { x: -50.0, y: 100.0, z: 30.0 },
       velocity: { mag: 0, ux: 0.0, uy: 1.0, uz: 0.0 },
       contacts: [],
       target: null,
       target_name: "",
-      radar_receiver: null,
+      radar_receiver: { "radius": 25.0 },
```

Comment:
the universe in the sample JSON file is only 200km wide, so this makes the radar scanning radius 1/8th the size of the universe. Should we narrow it a little bit?

Reply:
So I tried to go off of feeling while playing the game, and I think I'm fine with narrowing this, but we need to keep a good balance because if a player decides to fly in any given direction, they shouldn't see the blackness of space for too long or it will give the feeling of an empty universe. I tried to set it up so that it doesn't seem so empty, would love to have you play it and see what you think.